presto-main/src/main/java/com/facebook/presto/util/SpatialJoinUtils.java

```diff
@@ -69,15 +81,23 @@ private SpatialJoinUtils() {}
     private static boolean isSupportedSpatialFunction(FunctionCall functionCall)
     {
         String functionName = functionCall.getName().getSuffix();
-        return functionName.equalsIgnoreCase(ST_CONTAINS.getSuffix()) || functionName.equalsIgnoreCase(ST_WITHIN.getSuffix())
-                || functionName.equalsIgnoreCase(ST_INTERSECTS.getSuffix());
+        for (String allowedName : ALLOWED_SPATIAL_JOIN_FUNCTIONS) {
```

Comment:
why can't this be ALLOWED_SPATIAL_JOIN_FUNCTIONS.contains(functionName.toLower())?